### ISSUE TITLE
deno bundle: the bundle file does not have the same behavior as the source

### ISSUE THREAD

**Issue by oganexon** [denoland/deno#9450]
Let's consider these three files:

```ts
// three.ts
export let value = "old"

export function setValue() {
  value = "new";
}
```

```ts
// two.ts
export { value, setValue } from "./three.ts";
```

```ts
// one.ts
import { value, setValue } from "./two.ts";

setValue();
console.log(value)
```

If we run `one.ts`:
```shell script
$ deno run one.ts 
new
```

If we run `bundle.js`:
```shell script
$ deno bundle one.ts bundle.js

$ deno run bundle.js
old
```

The result is different.

Bundled file:

```js
// bundle.js
let value = "old";
const value1 = value;
function setValue() {
    value = "new";
}
const setValue1 = setValue;
setValue1();
console.log(value1);
```

Tested on:
- deno 1.7.1
- Windows, Linux

**Comment by kitsonk** [comment]
cc/ @kdy1

**Comment by bartlomieju** [comment]
Fixed in 1.8
